app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,10 @@
     <meta-data
       android:name="google_analytics_adid_collection_enabled"
       android:value="false" />
+    <meta-data
```

Comment:
The build started complaining unless I added this. I had to poke around in the merged manifest to figure out what went wrong. I tested a build to make sure analytics are still getting pushed, and it seems like it's all working fine.